### ISSUE TITLE
Out of date information

### ISSUE THREAD

**Issue by drelliche** [arcus/Arcus_Labs_Orientation#66]
This list has been compiled as part of the transition to having "Internet Disabled" be the default mode for labs.

New material will need to be added, but this issue focuses on existing material that will need to be updated.

- [ ] **Arcus Lab Basics** 
     - image needs to be updated
     - clean room paragraph should address internet on/off as a part of keeping the room clean
   
- [ ] **Included tools** Statistical Analysis Software now does include the option of some point and click tools (Stata, PSPP, Spreadsheets)
- [ ] **Arcus Scientific Project Request Stages** Use new graphics here instead of or in addition to the current mermaid diagram
- [ ] **Using and Arcus Lab** image needs to be updated
- [ ] **Lab Status Pane** 
     - image needs to be updated
     - add text description of "internet disabled" vs "internet enabled"
- [ ] **Project Members Pane**
     - Arcus staff members no longer appear in the project members pane
     - include links to tickets to [add](https://support.arcus.chop.edu/servicedesk/customer/portal/6/create/259?q=user&q_time=1722527654548) and ?remove? users 
- [ ] **Education Resources Pane**
     - Include the "Getting Started?" and "Need Extra Support?" panels
     - Update videos as appropriate
- [ ] **Version Control** 
     - Elaborate on the difference between using Git locally for version control and GitHub
     - Emphasize that pushing to GitHub (and pulling from GitHub) break the cleanroom.
     - Show how to use the new tools to clone a repository to your lab.

**Comment by drelliche** [comment]
@rosemm I think that the Version Control section outlined here is actually new content that is part of EDU-1741. Does that match your understanding?

Everything else has been added to this branch: https://github.com/arcus/Arcus_Labs_Orientation/tree/_internet_disabled_dev

**Comment by rosemm** [comment]
@drelliche yes, that's correct! And most of it is actually already covered in the existing New to GitHub, but that's not linked to in the main doc in an obvious place --- I'll put that in, and the other piece (show how to use the new tools to clone) is in https://github.com/arcus/Arcus_Labs_Orientation/pull/68
